brianlong/www.validators.app
# 187735523 fix max mind

#### What's this PR do?
- Updates maxmind scripts

#### How should this be manually tested?
- run append_data_centers_geo_data.rb and check_unknown_data_centers_for_updates.rb and review errors and results
- check number of validator ips with is_muted attribute

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187735523)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
